### ISSUE TITLE
Need recipes for chickens.

### ISSUE THREAD

**Issue by MisterShawn** [Drakma/VacantSky#20]
Many of the resource chickens from the Chickens mod are biome specific, but it's almost impossible to gather them in a skyblock. We need recipes to craft the flint, sand, log, and nether quartz chickens.

**Comment by Drakma** [comment]
They do spawn naturally.  I've gotten all 4 of those by building large areas of dirt.  That being said, I'll test the spawn rates and make recipes if I feel they are too low.

**Comment by kulflame** [comment]
I've had no issues spawning the chickens, just gotta be far enough away from the area.

**Comment by Drakma** [comment]
What he said.  Still I'll look at spawn rates.  I'm not opposed to recipes.  I just want them to be fair.

**Comment by MisterShawn** [comment]
Darkosto mentioned that he created recipes for the chickens for Sky Factory 3, so I assumed that their spawning was still biome-specific. I've only played for a short while and haven't gotten to spawning animals yet. I apologize for not checking if they spawned naturally.

**Comment by Drakma** [comment]
No worries.  Again, I'm not opposed to adding them if needed

**Comment by AstroxV** [comment]
I just finished my breeding line, got all chickens spawned on a grass plane some distance from my "crafting area". The only chicken to get from the nether was the nether quartz chicken, and there are loads of those in the nether, maybe a bit too many but its ok. I'm not sure if chicken bait will spawn the tier 1 chickens, I tried it in the beginning without luck.
I don't think a crafting recipe is needed especially with the hatchery mod, which makes it very easy to breed the chickens together and hatch the eggs. Maybe a little too easy as its just a matter of time with a tiny bit of automation. Throwing the eggs to hatch them would be fine too i think considering you get access to materials really hard to get otherwise.
